src/ReactiveUI.Validation/Helpers/ValidationHelper.cs

```diff
@@ -79,9 +79,9 @@ protected virtual void Dispose(bool disposing)
         private void Setup()
         {
             _disposables.Add(_validation.ValidationStatusChange.Select(v => v.IsValid)
-                .ToProperty(this, vm => vm.IsValid, out _isValid));
+                .BindTo(this, vm => vm.IsValid));
```

Comment:
Could you please provide more details about how this change should affect https://github.com/reactiveui/ReactiveUI.Validation/issues/19? From what I'm seeing right now, a `NullReferenceException` is going to be thrown when someone tries to access the `IsValid` and `Message` properties, because `_isValid` and `_message` fields stay uninitialized now.

Reply:
Sorry, but I can't open the project with visual studio 2017 and I had to do it without trying. What I want is for the IsValid property to spread and not for an exception, which is what happens right now. Changing it to BindTo works correctly.